AlphaWallet/InCoordinator.swift

```diff
@@ -231,11 +235,6 @@ class InCoordinator: NSObject, Coordinator {
         }
     }
 
-    private func setUpEventSourceCoordinatorForActivities() {
-        guard Features.isActivityEnabled else { return }
-        eventSourceCoordinatorForActivities = EventSourceCoordinatorForActivities(wallet: wallet, config: config, tokensDataStore: tokensDataStore, assetDefinitionStore: assetDefinitionStore, eventsDataStore: eventsActivityDataStore)
```

Comment:
Has this been moved elsewhere previously?

Reply:
this is my fail i missed to update RPs message for prev PR

Reply:
No worries, I might have seen it or missed it. Just wanted to make sure it's still there without testing it myself :)